src/barplot.ts

```diff
@@ -247,6 +247,15 @@ export function drawLabels(size:number[], data:INumericalVector, labels: d3.Sele
     'display': (rowHeight >= 15) ? 'inline' : 'none',
     'font-size': '14px'
   });
+  const categories = (<any>data.desc.value).categories;
+  const nameToLabel = new Map<string, string>();
+  if (categories) {
```

Comment:
But data is of type `INumericalVector` => `desc` is of type` IVectorDataDescription<INumberValueTypeDesc>` and` value `of type `INumberValueTypeDesc`. `InumberValueTypeDesc` has no member `categories`.